addons/purchase/models/purchase.py

```diff
@@ -127,6 +126,85 @@ def _compute_invoice(self):
     company_id = fields.Many2one('res.company', 'Company', required=True, index=True, states=READONLY_STATES, default=lambda self: self.env.company.id)
     currency_rate = fields.Float("Currency Rate", compute='_compute_currency_rate', compute_sudo=True, store=True, readonly=True, help='Ratio between the purchase order currency and the company currency')
 
+    reminder_confirmed = fields.Boolean("RFQ Confirmed", default=False, readonly=True, help="True if the reminder email is confirmed by the vendor.")
+    po_confirmed = fields.Boolean("PO Confirmed", default=False, readonly=True, help="True if PO reception is confirmed by the vendor.")
```

Comment:
po_confirmed is too broad

mail_reminder_confirmed
mail_reception_confirmed
?